samples/tfx-oss/README.md

```diff
@@ -0,0 +1,53 @@
+# TFX Pipeline Example
+
+This sample walks through running [TFX](https://github.com/tensorflow/tfx) [Taxi Application Example](https://github.com/tensorflow/tfx/tree/master/examples/chicago_taxi_pipeline) on Kubeflow Pipelines cluster. 
+
+## Overview
+
+This pipeline demonstrates the TFX capablities at scale. The pipeline uses a public BigQuery dataset and uses GCP services to preprpocess data (Dataflow) and train the model (Cloud ML Engine). The model is then deployed to Cloud ML Engine Prediction service.
+
+
+## Setup
+
+Create a local Python 3.5 conda environment
+```
+conda create -n tfx-kfp pip python=3.5.3
+```
+then activate the environment.
+
+
+Install TFX and Kubeflow Pipelines SDK
+```
+!pip3 install https://storage.googleapis.com/ml-pipeline/tfx/tfx-0.12.0rc0-py2.py3-none-any.whl 
+!pip3 install https://storage.googleapis.com/ml-pipeline/release/0.1.10/kfp.tar.gz --upgrade
```

Comment:
JFYI: The latest version is 0.1.11 and in 0.1.12 will have improved experiment creation.